docs/send-finality-vote.md

```diff
@@ -0,0 +1,82 @@
+# Finality Votes Submission Specification
+
+## Overview
+
+Finality providers submit votes to finalize blocks on the consumer chain.
+This document specifies the process of submitting finality votes.
+
+## Internal State
+
+The finality provider maintains two critical persistent states:
+
+- Last voted height
+- Last processed height
+
+### Last voted height
+
+Tracks the most recent height for which a finality vote was successfully
+submitted. This prevents duplicate voting on previously voted heights.
+
+### Last processed height
+
+Tracks the most recent height for which a voting decision was made
+(whether the decision was to vote or not).
+By definition, `LastProcessedHeight >= LastVotedHeight` always holds.
+
+## Submission Process
+
+### Bootstraping
+
+To determine the initial processing height:
+
+1. Query consumer chain for `lastFinalizedHeight`
+2. If no finalized blocks exist:
+   - Start from `finalityActivationHeight`
```

Comment:
there is a problem here.

imagine there is no finalized block, but this FP already voted for the first activated block and it's just waiting for the rest FP to vote so the block can get finalized

starting at `finalityActivationHeight` means a duplicate vote

Reply:
Good point. See updates in [ab8913a](https://github.com/babylonlabs-io/finality-provider/pull/120/commits/ab8913acdc3e03e7ad36aebad133075942916131)